Code/GraphMol/ForceFieldHelpers/CrystalFF/TorsionPreferences.cpp

```diff
@@ -0,0 +1,674 @@
+//
+//  Copyright (C) 2015 Sereina Riniker
+//
+//   @@ All Rights Reserved @@
+//  This file is part of the RDKit.
+//  The contents are covered by the terms of the BSD license
+//  which is included in the file license.txt, found at the root
+//  of the RDKit source tree.
+//
+#include <GraphMol/RDKitBase.h>
+#include <Geometry/Utils.h>
+#include <GraphMol/SmilesParse/SmilesParse.h>
+#include <GraphMol/Substruct/SubstructMatch.h>
+#include <RDGeneral/utils.h>
+#include <RDGeneral/RDLog.h>
+#include <RDGeneral/Exceptions.h>
+#include <boost/dynamic_bitset.hpp>
+#include <algorithm>
+#include <iostream>
+#include <sstream>
+#include <RDGeneral/StreamOps.h>
+
+#include <boost/lexical_cast.hpp>
+#include <boost/tokenizer.hpp>
+typedef boost::tokenizer<boost::char_separator<char> > tokenizer;
+#include <boost/flyweight.hpp>
+#include <boost/flyweight/key_value.hpp>
+#include <boost/flyweight/no_tracking.hpp>
+
+namespace ForceFields {
+  namespace CrystalFF {
+    using namespace RDKit;
+
+    typedef boost::dynamic_bitset<> BIT_SET;
```

Comment:
Why use this typedef? It doesn't seem to save that much typing but it does require people reading the code to realize that a BIT_SET is a boost::dynamic_bitset.